src/languages/es.ts

```diff
@@ -245,6 +245,14 @@ export default {
         recent: 'Reciente',
         all: 'Todo',
     },
+    location: {
+        useCurrent: 'Usar ubicación actual',
+        notFound: 'No pudimos encontrar tu ubicación, inténtalo de nuevo o introduce una dirección manualmente.',
+        permissionDenied: 'Parece que has denegado el permiso a tu ubicación.',
+        please: 'Por favor,',
```

Comment:
I see there is a `,` here, but not in the English translation. Is that intentional or an oversight?

Reply:
Yes @pecanoro confirmed that translation. The message @huzaifa-99 linked in the above comment is a copy from the original message below

<img width="300" alt="Screenshot 2023-09-20 at 7 28 36 PM" src="https://github.com/Expensify/App/assets/98560306/e485ddc6-877a-4b6c-93d8-2e2e94d9ff33">

Reply:
Is "Your Location" text saved in the list currently-used of addresses?

Reply:
No. We specifically filter out "Your Location" both on the front-end and on the back-end for the recently used addresses. I'd like to avoid adding a bunch of language checks to those filters (ie. like when we have 20+ languages, it's not going to scale). That could be a problem for later though.

Reply:
and we're using the text value to filter out? @huzaifa-99  Is thee another way to determine if a value is "Current Location" or not?

Reply:
Neil and I had 1:1 discussion about this.

let's not translate it at this point.

Addresses are not translated anyway, so we can leave "Current Location" as it is today. Button text is still translated right?

Reply:
perfect thank you for confirmation